src/types/app_remote_config_environment.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// AppRemoteEnvironment represents represents configuration for a particular environment
+type AppRemoteEnvironment struct {
+	Environment       map[string]string
```

Comment:
I think we should open an issue renaming this to EnvironmentVariables, here and everywhere else we use env vars

Reply:
Sounds good

Reply:
done